core/src/main/java/org/apache/iceberg/TableMetadata.java

```diff
@@ -146,6 +151,7 @@ static TableMetadata newTableMetadata(
         .setDefaultSortOrder(freshSortOrder)
         .setLocation(location)
         .setProperties(properties)
+        .setRowLineage(rowLineage)
```

Comment:
We don't want to set RowLineage to false if it is unset. So we have 2 options here

1) Set it to false but only if the table is V3
2) Set it to true only if it is true and leave it absent otherwise.

Reply:
You actually don't have to do anything! I forgot I already did this in "setRowLineage". It won't set the field unless it's getting set to true.

Reply:
@RussellSpitzer If you don't set  `row-lineage` in the table properties, it's unset, I mean it's kept empty because `setRowLineage` can handle null argument such as:

https://github.com/apache/iceberg/blob/bcbbd0344623ffea5b092e2de5debb0bc12892a1/core/src/main/java/org/apache/iceberg/TableMetadata.java#L1525-L1528